test/check-application

```diff
@@ -1417,6 +1417,11 @@ class TestApplication(testlib.MachineCase):
         b.click("#containers-containers button.pf-c-button.pf-m-primary")
         b.set_input_text("#run-image-dialog-name", container_name)
 
+        # Test invalid input
+        b.set_input_text("#create-image-image-select-typeahead", "*\\|?")
+        b.click('button.pf-c-toggle-group__button:contains("Local")')
+        b.wait_text("button.pf-c-select__menu-item.pf-m-disabled", "No images found")
```

Comment:
This doesn't prove much, though. Could this actually create an image with a ? or * in it, and make sure it gets found?

Reply:
Ah, haven't considered that, my main concern was that we wouldn't oops. What I would really like is to show an validation error when entering non-allowed characters. But this is a bit tricky with the select when the dropdown is shown above the validation line of PF.

Reply:
Is there some spec what kind of characters can appear in container names? If so, that would help indeed. 

Indeed docker seems to be rather  picky about it:
```
❱❱❱ podman tag docker.io/library/debian:unstable 'localhost:foo?bar'    
Error: normalizing name "localhost:foo?bar": normalizing name "localhost:foo?bar": invalid reference format    
❱❱❱ podman tag docker.io/library/debian:unstable 'localhost:foo*bar'    
Error: normalizing name "localhost:foo*bar": normalizing name "localhost:foo*bar": invalid reference format    
❱❱❱ podman tag docker.io/library/debian:unstable 'localhost:foo|bar'    
Error: normalizing name "localhost:foo|bar": normalizing name "localhost:foo|bar": invalid reference format    
❱❱❱ podman tag docker.io/library/debian:unstable 'localhost:foobär'    
Error: normalizing name "localhost:foobär": normalizing name "localhost:foobär": invalid reference format    
```

Reply:
The restriction seems to be this regex https://github.com/moby/moby/blob/be97c66708c24727836a22247319ff2943d91a03/daemon/names/names.go#L6

Reply:
This is about filtering containers, not about creating. So you can't actually select an invalid container image as those should not be returned from the registry (that would be a bug imo)

Reply:
Can this also test a string with some valid and some invalid characters? In this check, *all* characters are dropped.